samigo/samigo-app/src/webapp/jsf/event/eventLog.jsp

```diff
@@ -47,17 +48,17 @@
      </h:selectOneMenu>
    </div>
    <div class="divRight col-md-8 col-lg-6 col-sm-7 col-xs-12">
-      <h:inputText id="IE_hidden" value="" disabled="true" style="display: none;" />
-      <h:inputText id="filteredUser" value="#{eventLog.filteredUser}" size="30" autocomplete="off"
-         onfocus="resetHelpValue('#{eventLogMessages.search_hint}', 'eventLogId:filteredUser')"
-         onclick="resetHelpValue('#{eventLogMessages.search_hint}', 'eventLogId:filteredUser')"/>
+      <h:outputLabel value="#{eventLogMessages.search}"/>
+      <h:outputText value="&#160;" escape="false" />
+      <h:inputText id="filteredUser" value="#{eventLog.filteredUser}" style="width: 200px; height: 33px;" autocomplete="off"
```

Comment:
Rather than inline styles, you can use the "sakai-table-toolBar" styles that were introduced in SAK-40248 and used in SAK-40098. 

Examples of these styles are in Worksite Setup, Announcements, and Assignments for their main pages with tables using search, filters, and pagers. 

Essentially, you have a .sakai-table-toolBar container, then a .sakai-table-filterContainer, then view filters go in a .sakai-table-viewFilter div and search goes in a .sakai-table-searchFilter div. 

This structure aligns the elements properly without custom styles and adapts to multiple screen sizes automatically.